api/src/main/resources/messages.properties

```
@@ -243,6 +243,20 @@ ${project.parent.artifactId}.report.patients.datasetDescription=
 ${project.parent.artifactId}.report.drc.patients.reportName=DRC Patients Report
 ${project.parent.artifactId}.report.drc.patients.datasetName=REGISTRE DU SIDA AVANCE
 
+${project.parent.artifactId}.report.drc.hivStage.reportName=DRC HIV Stage 3/4 Report
+${project.parent.artifactId}.report.drc.hivStage.reportDescription=DRC HIV Stage 3/4 Report
+${project.parent.artifactId}.report.drc.hivStage.males.label=Males
```

Just to have symmetry between the English and French
```suggestion

${project.parent.artifactId}.report.drc.hivStage.males.label=Males
```